Pfofile Page component

Acid-base opened the pull request:
Built out most the page, I filled in placeholders and mock data where we're waiting on api.  Tried to comment decently well to explain my work

Acid-base committed:
Pfofile Page component

Initialize the concept, I filled in placeholders and mock data where we're waiting on api.  Tried to comment decently well to explain my thinking

Chad-Martinez commented on package.json:
```diff
@@ -10,8 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
```
Remove all of these added packages

Acid-base committed:
Revert "Pfofile Page component"

This reverts commit b49dbdd6d07e011c2b529fc58bd706c71bd6f8b4.

Acid-base committed:
Pfofile Page component

Initialize the concept, I filled in placeholders and mock data where we're waiting on api.  Tried to comment decently well to explain my thinking

Acid-base committed:
reverted old package.json

putting files back into upstream without the package.json conflict
